### ISSUE TITLE
Make VPNConnectionInfo accessible.

### ISSUE THREAD

**Issue by yzen** [mozilla-mobile/mozilla-vpn-client#136]
Right now connection information pane is not accessible.
There are a couple of things we can do to make it more delightful:
- internal `ChartView` must have `Accessible.name` and `Accessible.role` that describes the graph.
- Label that presents IP information must have `Accessible.name: text`, this way it will be available to screen readers as static text. It must also be in the right accessible order. Right now it seems to not be following the `Close` button when moving virtual cursor forward (I suspect it's simply because of the order of components inside `VPNConnectionInfo`).
- `VPNGraphLegendMarker` should have `Accessible.role` set to `Accessible.StaticText`. The accessible order seems to be off  as well, so we need to make sure that order of components matches the layout order (I suspect). It also must have `Accessible.name` for the same reasons as the IP label. It should be comprised of the label, value and range like this (granted the id's are added to the inner Text components):
```
//% "%1: %2 %3"
Accessible.name: qsTrId("vpn.connectionInfo.accessibleName") // or something similar
    .arg(label.text)
    .arg(value.text)
    .arg(range.text)
```
